### PR TITLE
Addons: Remove SDFGeometryGenerator.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: #29171

**Description**

Another example that I suggest to remove is `webgl_geometry_sdf`. When #26837 was merged, I have underestimated the dependency to GLSL and `WebGLRenderer`. Especially since with WebGPU you have compute shaders so something like `SDFGeometryGenerator` could be implemented in a different way.

Considering the complexity of `SDFGeometryGenerator` and that it is not common task to generate geometries from a SDF, it's best to maintain `SDFGeometryGenerator` in an external repository.

### CODE REVIEW

**Comment by mrdoob** [comment]
SGTM 👍

**Comment by santi-grau** [comment]
😿😿😿

**Comment by mrdoob** [comment]
Sorry, we can't maintain all the code 😕

File: examples/jsm/geometries/SDFGeometryGenerator.js [hunk 1]
```diff
@@ -1,144 +0,0 @@
...
-	Mesh,
-	OrthographicCamera,
-	PlaneGeometry,
-	Scene,
-	ShaderMaterial,
-	Vector2,
-	WebGLRenderTarget
-} from 'three';
-
-import { surfaceNet } from './../libs/surfaceNet.js';
```
Comment (mrdoob) [inline examples/jsm/geometries/SDFGeometryGenerator.js:21]: I think we forgot to remove `surfaceNet.js` too?

**Pull request merged**